spec/lib/vcr/cassette/persisters/file_system_spec.rb

```diff
@@ -64,12 +64,23 @@ class Persisters
 
             expected = File.join(FileSystem.storage_location, "\u842c\u570b\u78bc")
             expect(FileSystem.absolute_path_to_file("\u842c\u570b\u78bc")).to eq(expected)
+
+            expected = File.join(FileSystem.storage_location, "Uppercase_Cassette.yml")
+            expect(FileSystem.absolute_path_to_file("Uppercase_Cassette.yml")).to eq(expected)
           end
 
           it 'handles files with no extensions (even when there is a dot in the path)' do
             expected = File.join(FileSystem.storage_location, "/foo_bar/baz_qux")
             expect(FileSystem.absolute_path_to_file("/foo.bar/baz qux")).to eq(expected)
           end
+
+          it 'downcases cassette names if the option is passed' do
+            VCR.configuration.default_cassette_options.merge!(
+              { :persister_options => { :downcase_cassette_names => true }})
```

Comment:
The parenth linting here is bugging me out, would expect the closing one to be on a newline.